cypress/TestCases/FireboltCertification/Accessibility.feature

```diff
@@ -9,54 +9,54 @@ Feature: Accessibility
+         | Set textAlign-right                  | set textAlign to right                    | right for textAlign in closedcaptions settings                  | onclosedCaptionsSettingsChanged with right for textAlign                  | set textAlign to right                    |
+         | Set textAlignVertical-top            | set textAlignVertical to top              | top for textAlignVertical in closedcaptions settings            | onclosedCaptionsSettingsChanged with top for textAlignVertical            | set textAlignVertical to top              |
+         | Set textAlignVertical-middle         | set textAlignVertical to middle           | middle for textAlignVertical in closedcaptions settings         | onclosedCaptionsSettingsChanged with middle for textAlignVertical         | set textAlignVertical to middle           |
+         | Set textAlignVertical-bottom         | set textAlignVertical to bottom           | bottom for textAlignVertical in closedcaptions settings         | onclosedCaptionsSettingsChanged with bottom for textAlignVertical         | set textAlignVertical to bottom           |
+         | Set windowColor-white                | set windowColor to white                  | white for windowColor in closedcaptions settings                | onclosedCaptionsSettingsChanged with white for windowColor                | set windowColor to white                  |
+         | Set windowOpacity-50                 | set windowOpacity to 50                   | 50 for windowOpacity in closedcaptions settings                 | onclosedCaptionsSettingsChanged with 50 for windowOpacity                 | set windowOpacity to 50                   |
+         | Set preferredLanguages               | set preferredLanguages to spanish english | spanish for preferredLanguages in closedcaptions settings       | onclosedCaptionsSettingsChanged with spanish for preferredLanguages       | set preferredLanguages to spanish english |
 
    @Accessibility @coreSDK @sdk @transport
    Scenario Outline: Accessibility.closedCaptionsSettings - Positive Scenario: <Scenario> with 'null' params
       When '3rd party app' registers for the 'accessibility onClosedCaptionsSettingsChanged' event using the 'Firebolt' API
       And '3rd party app' invokes the 'Firebolt' API to 'get closedCaptions settings'
       And 1st party app invokes the 'Firebolt' API to '<Key>'
-      Then 'Firebolt' platform responds to '1st party app' with '<Set_Method_Content>'
+      Then 'Firebolt' platform responds to '1st party app' for '<Set_Method_Content>'
       When '3rd party app' invokes the 'Firebolt' API to 'get closedCaptions settings'
-      Then 'Firebolt' platform responds with '<Method_Content>'
+      Then 'Firebolt' platform responds for '<Method_Content>'
```

Comment:
Line 57 has to be changed to to use the same key as the setter call which is in line 56

Reply:
updated